tensorhive/models/User.py

```diff
@@ -25,6 +25,7 @@ class User(CRUDModel, Base):
     __tablename__ = 'users'
     id = Column(Integer, primary_key=True, autoincrement=True)
     username = Column(String(40), unique=True, nullable=False)
+    email = Column(String(40), unique=True, nullable=False)
```

Comment:
Does the e-mail have to be unique? Maybe I would like to have two accounts (for example one admin and one normal for testing).

Reply:
Ok, u're right